apps/console/src/components/pages/protected/trust-center/documents/documents-page.tsx

```diff
@@ -0,0 +1,111 @@
+'use client'
+
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import { DataTable } from '@repo/ui/data-table'
+import { Loading } from '@/components/shared/loading/loading'
+import { trustCenterDocsColumns, TTrustCenterDoc } from './table-config'
+import { VisibilityState } from '@tanstack/react-table'
+import { TPagination } from '@repo/ui/pagination-types'
+import { DEFAULT_PAGINATION } from '@/constants/pagination'
+import { useGetTrustCenterDocs } from '@/lib/graphql-hooks/trust-center'
+import DocumentsTableToolbar from './documents-table-toolbar'
+import { CreateDocumentSheet } from './create-document.sheet'
+import { TrustCenterDocWhereInput } from '@repo/codegen/src/schema'
+import { Panel, PanelHeader } from '@repo/ui/panel'
+import { Button } from '@repo/ui/button'
+import { File } from 'lucide-react'
+import Link from 'next/link'
+import { useRouter } from 'next/navigation'
+import { BreadcrumbContext } from '@/providers/BreadcrumbContext'
+
+const DocumentsPage = () => {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [columnVisibility, setColumnVisibility] = useState<VisibilityState>({ createdAt: false })
+  const [pagination, setPagination] = useState<TPagination>(DEFAULT_PAGINATION)
+  const [filters, setFilters] = useState<TrustCenterDocWhereInput | null>(null)
+  const router = useRouter()
+  const { setCrumbs } = useContext(BreadcrumbContext)
+
+  const { docs, paginationMeta, isLoading } = useGetTrustCenterDocs({
+    where: {
+      ...(searchTerm ? { titleContainsFold: searchTerm } : {}),
+      ...(filters ?? {}),
+    },
+    pagination,
+  })
+
+  const handleFilterChange = useCallback((newFilters: TrustCenterDocWhereInput) => {
+    setFilters(newFilters)
+    setPagination(DEFAULT_PAGINATION)
+  }, [])
+
+  const tableData =
+    docs.map((doc) => ({
+      id: doc?.id ?? '',
+      title: doc?.title ?? '',
+      category: doc?.category ?? '',
+      visibility: doc?.visibility ?? '',
+      tags: doc?.tags ?? [],
+      createdAt: doc?.createdAt ?? '',
+      updatedAt: doc?.updatedAt ?? '',
+    })) ?? []
+
+  const handleRowClick = (row: TTrustCenterDoc) => {
+    router.push(`/trust-center/documents?id=${row.id}`)
+  }
+
+  useEffect(() => {
+    setCrumbs([
+      { label: 'Home', href: '/dashboard' },
+      { label: 'Trust Center Documents', href: '/trust-center/documents' },
+    ])
+  }, [setCrumbs])
+
+  if (isLoading) return <Loading />
+  const hasData = tableData.length > 0
+
+  return (
+    <>
+      <CreateDocumentSheet />
+      {hasData ? (
+        <div className="p-4">
+          <h2 className="text-xl font-semibold mb-4">Trust Center Documents</h2>
```

Comment:
```suggestion
          <h2 className="text-xl font-semibold mb-4">Reports & Certifications</h2>
```

Can you also add some info/sub text that says:
```
Provide your customers with verified documentation such as SOC 2 reports, ISO 27001 certificates, or penetration test summaries that reinforce your organization’s security and compliance commitments.
```